### ISSUE TITLE
Add an “inactive” metadata annotation

### ISSUE THREAD

**Issue by kwatsen** [netmod-wg/yang-next#9]
The conditional-enablement draft seems to have support, but no one wants to rev NC/RC for it, so adding it into a rev of YANG might be the best option...

**Comment by schoenw** [comment]
Why would that have to be part of YANG 1.x?

**Comment by kwatsen** [comment]
unsure if this is a YANG thing or not

**Comment by kwatsen** [comment]
this could potentially be implemented via the introduction of "critical" extension statements

**Comment by mbj4668** [comment]
I don't think this a YANG language issue.  It can be seen as protocol-specific issue, or a meta-model issue.

**Comment by kwatsen** [comment]
Better implemented via critical annotation / module

**Comment by abierman** [comment]
agree this is not part of yang-next.
IMO protocol work should not be done either

**Comment by kwatsen** [comment]
Close here, after opening in proto-req tracker

more a proto-binding document.
